3party/jansson/myjansson.cpp

```diff
@@ -0,0 +1,79 @@
+#include "3party/jansson/myjansson.hpp"
+
+namespace my
+{
+void FromJSON(json_t * root, string & result) { result = string(json_string_value(root)); }
```

Comment:
I know it's not your code, but it would be nice to throw exception here if root is not a string.